packages/libllvm/clang.subpackage.sh

```diff
@@ -9,26 +10,31 @@ bin/*g++
 bin/*gcc
 bin/git-clang-format
 bin/hmaptool
+bin/intercept-build
 bin/modularize
 bin/pp-trace
 bin/scan-*
 include/clang*
 include/omp*.h
 lib/clang/*/include/*.h
 lib/clang/*/include/*.modulemap
+lib/clang/*/include/cuda_wrappers/
 lib/clang/*/include/openmp_wrappers/
 lib/clang/*/include/ppc_wrappers/
-lib/clang/*/include/cuda_wrappers/
 lib/cmake/clang
+lib/cmake/openmp
 lib/libclang*so
+lib/libear/
 lib/libomp.a
+lib/libscanbuild/
 libexec/
 share/clang
-share/scan-*
 share/man/man1/clang.1.gz
 share/man/man1/scan-*
+share/scan-*
 "
 TERMUX_SUBPKG_DESCRIPTION="C language frontend for LLVM"
+TERMUX_SUBPKG_DEPENDS="binutils, ndk-sysroot, libcompiler-rt"
```

Comment:
clang 13 switches to linking against compiler-rt and libunwind by default, llvm/llvm-project@91f8aacc0, so I made the former a dependency and added the latter to `ndk-sysroot`.

Reply:
Hmm, I got `libunwind.a` from NDK 23 and didn't check NDK 21d, which apparently only had it for arm.

@Grimler91, are we switching to NDK 23 soon?